charts/openshift-etcd-backup/templates/prometheusrule.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and (.Values.monitoring.enabled) (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "openshift-etcd-backup.fullname" . }}-cronjob-monitor
+spec:
+  groups:
+  - name: openshift-etcd-backup
+    rules:
+    - alert: EtcdBackupCronJobStatusFailed
+      expr: |
+        kube_job_status_succeeded{namespace="{{ .Release.Namespace }}"} == 0
+      labels:
+        severity: critical
```

Comment:
please move the actual definition to values.yaml (like [how the argo-cd helm chart does it](https://github.com/argoproj/argo-helm/blob/e4f8dd58109eefe53c41acd98ab50fd443643b22/charts/argo-cd/values.yaml#L188-L190))

This way tuning the rule or adding more rules is much easier.

Reply:
@hairmare i dont like this at all, other charts doing it as @vmaillot did here

Reply:
I'm having mixed feelings about that. I'd rather go the way of `kube-prometheus-stack` and how the handle it with the [`defaultRules`](https://github.com/prometheus-community/helm-charts/blob/2cacc16807caedc6cabf1606db27e0d78c844564/charts/kube-prometheus-stack/values.yaml#L33) parameter.

Reply:
that looks neat! i'll try with that solution!

Reply:
`kube-prometheus-stack` also supports `additionalPrometheusRulesMap` to add more rules (which is what i was after)

Reply:
For me this is perfectly fine. But i think what @hairmare was after is an `additionalRules` for adding more extra rules. But we should keep it simple and not forget what this is, its not like this chart is deploying a full prom stack or argocd as in the examples, it just one single pod cronjob wich runs periodically.

Reply:
yeah. i want to be able to add additional rules so we can validate them for a while before we add them directly in here... one rule that comes to mind would check `kube_cronjob_spec_suspend` to alert us if the job gets suspended

Reply:
just to be sure, should i add the support of [this](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/additionalPrometheusRules.yaml)?

Reply:
it doesn't need to be as complicated as the upstream kps chart does it, but it sure would be nice to be able to add a simple rule for checking `kube_cronjob_spec_suspend` on the fly where needed.